course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -106,14 +245,41 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(e1 * e1.coord('latitude'))"
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.1d"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Cube broadcasting is also taking place, meaning that the two inputs (cube, coordinate, array, or even constant value) don't need to have the same shape:"
+    "----\n",
+    "\n",
+    "Another function of cube arithmetic is to support 'broadcasting', in the numpy sense :  operations between data with different shapes.\n",
+    "\n",
+    "In fact we already saw this above, with `product = e1 * e1.coord('latitude')`.\n",
+    "\n",
+    "Broadcasting is simpler in Iris than in numpy, because how the dimensions \"line up\" is determined by matching coordinates, rather than depending on the ordering of dimensions.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Exercise: </font></b>\n",
+    "    <p>The basic result values in the above example,\n",
```

Comment:
The word basic here makes this phrasing a bit confusing.
Could we perhaps change this to
```The result values```
or ```The output values```
or ```The output data values```